.github/workflows/az-e2e.yaml

```diff
@@ -0,0 +1,34 @@
+name: azure e2e tests
```

Comment:
We already have [enclave-cc e2e tests](https://github.com/confidential-containers/operator/actions/workflows/e2e.yaml) workflow. What it will be testing on this workflow is the ccruntime implementation; so perhaps we can name it `ccruntime e2e tests` instead of `azure e2e tests`. Or, as we talked another day, name it `ccruntime functional tests`. The filename should be renamed properly too.

Reply:
ack, since the jenkins jobs will be gone it makes sense to avoid specifying az :-)